src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -159,20 +115,26 @@ public static Apartment jsonToApartment(Path jsonPath) throws IOException {
 	 * @throws IOException if the JSON file can't be created.
 	 */
 	public static void apartmentsToJson(List<Apartment> listApartments) throws IOException {
-		apartmentsToJson(listApartments, APARTMENT_PATH_JSON);
+		apartmentsToJson(listApartments, APARTMENTS_PATH_JSON);
 	}
 
 	/**
 	 * Converts a list of Apartments to a JSON file.
 	 *
 	 * @param listApartments <code>{@link List}</code> object to convert into JSON
+	 *                       (<b>listApartments</b> could contain only one
+	 *                       apartment)
 	 * @param jsonPath       <i>String</i> the path where to create the JSON file
 	 * @throws IOException if the JSON file can't be created.
 	 */
 	public static void apartmentsToJson(List<Apartment> listApartments, Path jsonPath) throws IOException {
```

Comment:
Après avoir testé, si deux appels sont fait pour écrire dans un même endroits, les données précédentes vont être écrasées. Serait-il possible de le préciser dans le contrat svp ?

Reply:
Fix in commit 4a413834bce85d7678953cb0b8b7fc08032f4799